src/hooks/useAddressHistory.ts

```diff
@@ -64,6 +64,12 @@ export function useAddressHistory() {
   };
 
   useEffect(() => {
+    if (
+      (!isAddressEmpty && !id) ||
+      (!id && window.location.pathname === ROUTES.jetton + "/") ||
+      window.location.pathname === ROUTES.jetton
+    )
+      showNotification("Invalid jetton address", "error");
```

Comment:
1. I don't think it belongs here, because this isn't address history functionality. Perhaps move this to the view jetton page.
2. Can we change `id` to `jettonAddress`? `id` doesn't seem like a good fit